src/ModelContextProtocol/Server/AIFunctionMcpServerTool.cs

```diff
@@ -207,35 +207,23 @@ public override async Task<CallToolResponse> InvokeAsync(
             },
             TextContent textContent => new()
             {
-                Content = [new() { Text = textContent.Text, Type = "text" }]
+                Content = [textContent.ToContent()]
             },
             DataContent dataContent => new()
             {
-                Content = [new()
-                    {
-                        Data = dataContent.GetBase64Data(),
-                        MimeType = dataContent.MediaType,
-                        Type = dataContent.HasTopLevelMediaType("image") ? "image" : "resource",
-                    }]
+                Content = [dataContent.ToContent()]
             },
```

Comment:
You can consolidate these now into just:
```C#
AIContent aiContent => new()
{
    Content = [aiContent.ToContent()]
},
```
If we improve ToContent in the future to handle other content types, this will get it for free. And for any others, the handling done by ToContent is the same as the default case.